src/main/kotlin/no/nav/klage/oppgave/config/problem/ProblemHandlingControllerAdvice.kt

```diff
@@ -0,0 +1,182 @@
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.NOT_FOUND, ex, request)
+
+    @ExceptionHandler
+    fun handleVedtakNotFound(
+        ex: VedtakNotFoundException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.NOT_FOUND, ex, request)
+
+    @ExceptionHandler
+    fun handleMeldingNotFound(
+        ex: MeldingNotFoundException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.NOT_FOUND, ex, request)
+
+    @ExceptionHandler
+    fun handleSaksdokumentNotFound(
+        ex: SaksdokumentNotFoundException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.NOT_FOUND, ex, request)
+
+    @ExceptionHandler
+    fun handleValidationException(
+        ex: ValidationException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.BAD_REQUEST, ex, request)
+
+    @ExceptionHandler
+    fun handleKlagebehandlingAvsluttetException(
+        ex: KlagebehandlingAvsluttetException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.FORBIDDEN, ex, request)
+
+    @ExceptionHandler
+    fun handleMissingTilgang(ex: MissingTilgangException, request: NativeWebRequest): ResponseEntity<Problem> =
+        create(Status.FORBIDDEN, ex, request)
+
+    @ExceptionHandler
+    fun handleNotMatchingUser(ex: NotMatchingUserException, request: NativeWebRequest): ResponseEntity<Problem> =
+        create(Status.FORBIDDEN, ex, request)
+
+    @ExceptionHandler
+    fun handleFeatureNotEnabled(ex: FeatureNotEnabledException, request: NativeWebRequest): ResponseEntity<Problem> =
+        create(Status.FORBIDDEN, ex, request)
+
+    @ExceptionHandler
+    fun handleNoSaksbehandlerRoleEnabled(
+        ex: NoSaksbehandlerRoleException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.FORBIDDEN, ex, request)
+
+    @ExceptionHandler
+    fun handleNotOwnEnhet(ex: NotOwnEnhetException, request: NativeWebRequest): ResponseEntity<Problem> =
+        create(Status.FORBIDDEN, ex, request)
+
+    @ExceptionHandler
+    fun handleResponseStatusException(
+        ex: WebClientResponseException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(ex, createProblem(ex), request)
+
+    @ExceptionHandler
+    fun handleDuplicateOversendelse(
+        ex: DuplicateOversendelseException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.CONFLICT, ex, request)
+
+    @ExceptionHandler
+    fun handleKlagebehandlingManglerMedunderskriverException(
+        ex: KlagebehandlingManglerMedunderskriverException,
+        request: NativeWebRequest
+    ): ResponseEntity<Problem> =
+        create(Status.BAD_REQUEST, ex, request)
+
+    @ExceptionHandler
+    fun handleKlagebehandlingFinalizedException(
```

Comment:
Jepp. Er litt å slette etterhvert.

Reply:
Ja, det er det. Jeg har ikke tatt for meg alle exceptionene ennå.